packages/nova-react/README.md

```diff
@@ -6,4 +6,71 @@ This package provides a React specific implementation of the contexts etc requir
 
 ## Prerequisites
 
-In order to use this package without compilation of documents, i.e. to parse string documents at runtime, make sure to add `@graphitation/graphql-js-tag` as a dependency in your `package.json` along side with `@nova/react`.
\ No newline at end of file
+In order to use this package without compilation of documents, i.e. to parse string documents at runtime, make sure to add `@graphitation/graphql-js-tag` as a dependency in your `package.json` along side with `@nova/react`.
+
+## Bundle size cost
+
+In order to eliminate majority of cost from the bundle, several pre-conditions must be met:
+
+- Your bundler, typescript and package should configured to use ESM/.mjs modules for module resolution
+- You should use a transform to parse graphql documents on compile time
+
+### Bundler and typescript configuration
+
+In your package, use `import` and `export` statements instead of `require`. In tsconfig.json your `module` should be set to `ES6`. Webpack 5 seems to be doing the right thing after the previous two items are done. For Webpack 4, things might not work in a weird way. We recommend updating to Webpack 5. One way we have seen people make it work in webpack 4 is by adding the following to their webpack config. YMMV.
```

Comment:
Have we really seen them getting it to work or were there still issues after making that change?

Reply:
It worked for them, but some other CI things stopped working. So I think it's a valid workaround.